.cargo/config

```diff
@@ -0,0 +1,2 @@
+[target.x86_64-unknown-linux-musl]
+linker = "x86_64-linux-musl-gcc"
```

Comment:
Is it needed to set `x86_64-unknown-linux-musl` in .cargo/config and as a environment variable?